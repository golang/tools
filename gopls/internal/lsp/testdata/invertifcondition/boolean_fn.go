package invertifcondition

import (
	"fmt"
	"os"
)

func BooleanFn() {
	if os.IsPathSeparator('X') { //@suggestedfix("if os.IsPathSeparator('X')", "refactor.rewrite", "")
		fmt.Println("A")
	} else {
		fmt.Println("B")
	}
}
