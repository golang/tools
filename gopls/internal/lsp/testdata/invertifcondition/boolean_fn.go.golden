-- suggestedfix_boolean_fn_9_2 --
package invertifcondition

import (
	"fmt"
	"os"
)

func BooleanFn() {
	if !os.IsPathSeparator('X') {
		fmt.Println("B")
	} else { //@suggestedfix("if os.IsPathSeparator('X')", "refactor.rewrite", "")
		fmt.Println("A")
	}
}

